src/scripts/autocomplete.js

```diff
@@ -0,0 +1,509 @@
+/*jslint indent: 2, unparam: true, plusplus: true*/
+/*global document: false, window: false, XMLHttpRequest: false, chrome: false, btoa: false, localStorage:false */
```

Comment:
maybe misunderstanding how this works; but `window`, `XMLHttpRequest`, `chrome`, `btoa` and `localStorage` not used in this file -> only need `/*global document: false */` ...?